### PR TITLE
Update fs2Instances package to ce3

### PR DESCRIPTION
**Pull request by FunFunFine**
This PR updates fs2 version and so that updates ce3 version.
I expect some other PR to do version update the right way, while this one focuses solely on the update of fs2Instances package.

Also this PR should target another branch specific for ce3-migration. I'll edit this when this branch is created.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ object Dependencies {
 
     val cats = "2.6.1"
 
-    val catsEffect = "2.5.1"
```
Comment (Odomontois) [inline project/Dependencies.scala:19]: lol

Comment (Odomontois) [reply project/Dependencies.scala:19]: No, that's wont work. You can wait for #720 where both ce dependencies will be in the build

File: project/Dependencies.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@ object Dependencies {
...
+    val catsEffect = "3.1.1"
```
Comment (Odomontois) [inline project/Dependencies.scala:19]: No, that's wont work. You can wait for #720 where both ce dependencies will be in the build

Comment (FunFunFine) [reply project/Dependencies.scala:19]: Quote from the description:
> I expect some other PR to do version update the right way

File: modules/fs2/src/main/scala/tofu/fs2/fs2Instances.scala [hunk 3]
```diff
@@ -62,38 +64,38 @@ private[fs2Instances] trait Fs2Instances1 extends Fs2Instances2 {
         ffa.parJoin(maxConcurrent)
     }
 
-  implicit def fs2PaceInstance[F[_]: Timer]: Pace[Stream[F, *]] =
+  implicit def fs2PaceInstance[F[_]: ceTemporal]: Pace[Stream[F, *]] =
```
Comment (Odomontois) [inline modules/fs2/src/main/scala/tofu/fs2/fs2Instances.scala:67]: using uncapitalized names for typeclasses is unusual
`TemporalCE`, mybe?

**Review by Odomontois — APPROVED** [review]

**Comment by Odomontois** [comment]
Close this for now

**Pull request closed without merging**